doc/source/reference/c-api/array.rst

```diff
@@ -634,8 +633,17 @@ From other objects
     requirements set to :c:data:`NPY_ARRAY_DEFAULT` and the type_num member of the
     type argument set to *typenum*.
 
-.. c:function:: PyObject *PyArray_FromObject( \
-        PyObject *op, int typenum, int min_depth, int max_depth)
+.. c:function:: PyObject* PyArray_ContiguousFromObject( \
+        PyObject* op, int typenum, int min_depth, int max_depth)
+
+    This function returns a well-behaved C-style contiguous array. The minimum
+    number of dimensions the array can have is given by `min_depth` while the
```

Comment:
The other related docstrings all mention what the array is created *from*, e.g., from the `PyArray_FromObject` docstring:

> Return an aligned and in native-byteorder array from any nested
> sequence or array-interface exporting object...

It might be worth including the "from any nested sequence or array-interface-exporting object" here for consistency.

Reply:
Done!